sdk/storage/storage-blob-changefeed/review/storage-blob-changefeed.api.md

```diff
@@ -4,12 +4,25 @@
 
 ```ts
 
-import { BlobServiceClient } from '@azure/storage-blob';
+import { AbortSignalLike } from '@azure/core-http';
+import { AnonymousCredential } from '@azure/storage-blob';
+import { CommonOptions } from '@azure/storage-blob';
 import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { Pipeline } from '@azure/storage-blob';
+import { StoragePipelineOptions } from '@azure/storage-blob';
+import { StorageSharedKeyCredential } from '@azure/storage-blob';
+import { TokenCredential } from '@azure/core-http';
 
 // @public (undocumented)
 export class BlobChangeFeedClient {
-    constructor(blobServiceClient: BlobServiceClient);
+    constructor(url: string, credential?: StorageSharedKeyCredential | AnonymousCredential | TokenCredential, options?: StoragePipelineOptions);
+    constructor(url: string, pipeline: Pipeline);
+    // (undocumented)
+    static appendUserAgentPrefix(options?: StoragePipelineOptions): StoragePipelineOptions;
+    // (undocumented)
+    static fromConnectionString(connectionString: string, options?: StoragePipelineOptions): BlobChangeFeedClient;
+    // Warning: (ae-forgotten-export) The symbol "BlobChangeFeedListChangesOptions" needs to be exported by the entry point index.d.ts
```

Comment:
Need to export `BlobChangeFeedListChangesOptions`?

Reply:
Fixed.